apps/table.html

```diff
@@ -267,6 +275,105 @@ <h5 class="modal-title" id="deleteModalLabel">Delete confirmation</h5>
 		</div>
 	</div>
 
+  <!-- Manage Collaboration Modal -->
+  <div class="modal fade" id="manageCollaborationModal" role="dialog" tabindex="-1" aria-labelledby="manageCollaborationModalLabel" aria-hidden="false">
+    <div class="modal-dialog modal-dialog-centered modal-lg" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title" id="manageCollaborationModalLabel">Manage Collaboration</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close" onclick="$('#manageCollaborationModal').modal('hide')">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        <div class="modal-body">
+          <b>Slide Name: <span id="modal-collab-slidename"></span></b>
```

Comment:
@Vedant1202  It shows Slide ID to me instead of slide name.

Reply:
Yeah, but not all slides are having names. I guess I'll add a check else return slide Id